tools/db_bench_tool.cc

```diff
@@ -1154,6 +1154,10 @@ DEFINE_string(simulate_hybrid_fs_file, "",
               "File for Store Metadata for Simulate hybrid FS. Empty means "
               "disable the feature. Now, if it is set, "
               "bottommost_temperature is set to kWarm.");
+DEFINE_int32(simulate_hybrid_hdd_multipliers, 1,
+             "In simulate_hybrid_fs_file or simulate_hdd mode, how many HDDs "
+             "are simulated.");
```

Comment:
should we explain a little bit more about hdd multiplier, that it increases the simulated IO throughput?

Reply:
Let me try to merge it first and and do it as a follow up.